packages/xml-views-tooltip/src/documentation.ts

```diff
@@ -0,0 +1,154 @@
+import {
```

Comment:
This file should be in logic-utils. Since you copied it as-is, please put a comment at the top that says where you copied it from, add an `istanbul ignore file` comment that says it will move to logic-utils and will be tested there soon.